docs/release_notes/release-2.3.md

```diff
@@ -1,5 +1,18 @@
 # StarRocks version 2.3
 
+## 2.3.7
+
+Release date: December 30, 2022
+
+### Bug fixes
+
+The following bugs are fixed:
+
+- A column named `a` in a StarRocks table is allowed to be NULL, but the column `a` in a view `t` that is created on that table is incorrectly set to be NOT NULL. [#15749](https://github.com/StarRocks/starrocks/pull/15749)
```

Comment:
The column that is allowed to be NULL in a StarRocks table is incorrectly set to NOT NULL in a view created from that table.